### PR TITLE
Satellites need to know all the Alignak objects...

### PR DESCRIPTION
**Pull request by mohierf**
... else it is not possible to unserialize poller/reactionne modules.

### CODE REVIEW

**Comment by Seb-Solon** [comment]
Can you provide an example ?

We have added such thing in the init file of bin lib (https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/bin/__init__.py). In what case we use satellite without a daemon?

**Comment by mohierf** [comment]
Example is very easy: simply add a module to a poller and you will have an AlignakClassLookupException exception when the poller tries to unserialize its modules ...

**Comment by Seb-Solon** [comment]
Yes, but what the stack trace is? Do you think, it is only a matter of adding the missing object into the __init__ file I mentioned?

**Comment by mohierf** [comment]
You got it :+1:

File: alignak/satellite.py [hunk 1]
```diff
@@ -995,16 +999,18 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
...
-            mods = unserialize(g_conf['modules'], True)
-            self.new_modules_conf = []
-            for module in mods:
-                # If we already got it, bypass
-                if module.python_name not in self.q_by_mod:
-                    logger.debug("Add module object %s", str(module))
-                    self.new_modules_conf.append(module)
-                    logger.info("[%s] Got module: %s ", self.name, module.python_name)
-                    self.q_by_mod[module.python_name] = {}
+            if not self.have_modules:
```
Comment (Seb-Solon) [inline alignak/satellite.py]: Why is this if needed?

Comment (mohierf) [inline alignak/satellite.py]: It is not necessary. I added this when testing but I will remove ...

**Comment by mohierf** [comment]
@Seb-Solon can you merge this PR please, else the nrpe-booster can not be used with Alignak ... :/P

**Comment by Seb-Solon** [comment]
NB : This PR remove some detail in debug logs to be discussed later #535

**Pull request merged**